e2e/generate_test.go

```diff
@@ -28,3 +28,49 @@ func TestGenerate_FullNode_GoerliNotSupported(t *testing.T) {
 	// Run test case
 	e2eTest.run()
 }
+
+func TestGenerate_FullNode_Lido_Sepolia_NoMEV(t *testing.T) {
+	// Test context
+	var (
+		runErr error
+	)
+	// Build test case
+	e2eTest := newE2ETestCase(
+		t,
+		// Arrange
+		nil,
+		// Act
+		func(t *testing.T, binaryPath string) {
+			runErr = runSedge(t, binaryPath, "generate", "full-node", "--lido", "--network", "sepolia", "--no-mev-boost")
+		},
+		// Assert
+		func(t *testing.T) {
+			assert.NoError(t, runErr, "generate command should succeed with the given arguments")
```

Comment:
We should think of a way to make more assertions and find out that the specific things related to the use of the `--lido` flag are there in the generated files. The E2E suite was added with an example but further code for asserting the right things from the `generate` cmd (and other commands) is missing. Be creative.